copter/source/docs/traditional-helicopter-connecting-apm.rst

```diff
@@ -10,63 +10,54 @@ Autopilot Info
 
 Overview of servo, and RX connection
 ====================================
 
-The RC input for many ardupilot compatible autopilots is either PPM SUM
-(8 channels) or S.Bus (up to 18 channels).  Some controllers also accept Spektrum
-satellite receivers.  If you have a receiver that outputs only PWM, you will need
-a PPM encoder to connect to your autopilot.
-
-The default RC input and SERVO output mapping is as follows:
-
-+--------------+-------------+
-| RC Channel   | SERVO output|
-+--------------+-------------+
-| 1            | Aileron     |
-+--------------+-------------+
-| 2            | Elevator    |
-+--------------+-------------+
-| 3            | Collective  |
-+--------------+-------------+
-| 4            | Rudder      |
-+--------------+-------------+
-| 5            | Flight Modes|
-+--------------+-------------+
-| 6            | Tuning      |
-+--------------+-------------+
-| 7            | Aux         |
-+--------------+-------------+
-| 8            | Throttle    |
-+--------------+-------------+
+The RC input for many ardupilot compatible autopilots is either PPM SUM (8 channels) or S.Bus (up to 18 channels).  Some controllers also accept Spektrum satellite receivers.  For receivers that only output PWM, a PPM encoder is required to connect to the autopilot.
+
+The default receiver channel to ArduCopter RC input function mapping is as follows:
+
++--------------+----------------+
+| RC Receiver  | Ardupilot RC   |
+| Channel      | Input Function |
++--------------+----------------+
+| 1 (Aileron)  | Roll*          |
++--------------+----------------+
+| 2 (Elevator) | Pitch*         |
++--------------+----------------+
+| 3 (Throttle) | Collective*    |
++--------------+----------------+
+| 4 (Rudder)   | Yaw*           |
++--------------+----------------+
+| 5 (Gear)     | Flight Mode    |
++--------------+----------------+
+| 6 (Aux 1)    | Tuning         |
++--------------+----------------+
+| 7 (Aux 2)    | Aux            |
++--------------+----------------+
+| 8 (Aux 3)    | Motor Interlock|
+|              | (throttle)     |
++--------------+----------------+
+* Functions are already mapped by RCMAP parameters. 
 
 The output pins on most controllers for SERVO's 1 thru 8 are labled Main Out:
 
 .. image:: ../images/PH21_2.jpg
     :target: ../_images/PH21_2.jpg
 
-Motor 1, which is normally the left front servo on your helicopter goes to pin 1.
-Motor 2, which is normally the right front servo goes to pin 2.  Motor 3, which 
-is normally the rear (elevator) servo goes to pin 3. Tail servo goes to pin 4.
+The :ref:`autopilot output functions wiki <common-rcoutput-mapping>` shows the complete list of servo output functions.  The default swashplate is H3-120 where Motor 1, left front servo, goes to pin 1; Motor 2, right front servo goes to pin 2; and Motor 3, rear (elevator) servo, goes to pin 3. See the :ref:`swashplate setup wiki <traditional-helicopter-swashplate-setup>` for more details.
```

Comment:
I think we should change all references from "pins" to "outputs"....might be good to re-iterate that Motor 3 corresponds to having SERVOn_FUNCTION = 35 where n is the output number.  Different autopilots have different marking for their outputs and some dont have "pins" marked at all...

Reply:
Yes, this is true. The outputs are SERVO. The inputs for aux ports are "pins".